sysutils/smart/src/www/diag_smart.php

```diff
@@ -302,6 +304,9 @@ function add_colors($string)
       </section>
 
     <?php
+    } else {
+        echo gettext("No SMART devices.");
+    }
```

Comment:
gettext() is better here, though not much use to retrofit all old cases. I don’t see a long future for this very old code base and very little benefit by modernising it.